### ISSUE TITLE
Shoutcast v2 and Station without Auto-DJ

### ISSUE THREAD

**Issue by renatopinto89** [AzuraCast/AzuraCast#4077]
Hello and first of all, congratulations on the project, this is a godsend, you guys are amazing.

Ok so here´s my issue. I´m trying to set up a station with ShoutCast V2, but I need that particular station to have Auto-DJ off and to be used only for live streams to save server resources and not have another instance of LiquidSoap running. The problem is, when i set up a station like that, the DJs in question that try to log into it, can only connect using RadioBoss, which has native support for shoutcast v.2.
In the case of VirtualDJ, Mixxx or SAM, it always gives a error saying it can´t connect to server. I´ve tried everything including the "password:#sid" and nothing works, only RadioBoss does work. 

I know there´s some authentication done through LiquidSoap, but if RadioBoss works, the others should too (I think). Is there anything I can do to fix it without it being to run Auto-DJ with a "silence.mp3" on repeat and set a broadcaster (works but far from ideal) or am I just doing something wrong?

Thank you

AzuraCast v0.12.2 Stable • Docker - Ubuntu 18

**Comment by Vaalyn** [comment]
If you are using the `DJ Account` feature you need to have Liquidsoap running.

If you give your DJs the main password for the ShoutCast server they should be able to directly connect to that.
Regarding ShoutCast V2 / V1 protocol, I think SC DNAS 2.x should still be able to accept V1 connections but that might be on a different port. Not sure though since I'm not using ShoutCast.

**Comment by renatopinto89** [comment]
It's the main source password. And yes ShoutCast V2 is legacy compatible, and it works if the same DNAS is running as standalone. But there's some kind of "denial" on AzuraCast that keeps the legacy encoders to connect to the DNAS, even if using the legacy port... Except RadioBoss... That one connects just fine.

**Comment by NadjaNeville** [comment]
@SpeedyCharly try putting a 1 in the stream ID field then username:password in the password field

**Comment by renatopinto89** [comment]
@NadjaNeville Neither VirtualDJ or Mixxx have the SID field. On the ones that do have that field (ShoutCast V2 compatible) it works without a problem. And about the username:password that feature is only applicable when you have Auto-DJ enabled on the station. Which in this particular case is not desirable. 
This is a very weird "bug" to have.

**Comment by renatopinto89** [comment]
@Vaalyn @SC2Mitch 
I finally figured it out. Since in order for a legacy SHOUTcast V1 encoder to connect to a SHOUTcast V2 server, it uses the port +1, and after looking at the docker-compose.yml, the port base for each station includes the main port and the two ports for Liquid soap (XXX5 and XXX6). On my specific case since I don't want to use Auto-DJ on a specific station, I just added the +1 port that I needed (or just set the station to use the Liquid Soap ports instead) and voila, it connects and everything works! Took me months but finally figured out haha!
I know this is a very specific example and probably useless for most but just wanted to share the "fix"

Happy streaming folks!
